hazelcast-client/src/main/java/com/hazelcast/client/spi/properties/ClientProperty.java

```diff
@@ -86,6 +86,20 @@
             = new HazelcastProperty("hazelcast.client.max.concurrent.invocations", Integer.MAX_VALUE);
 
     /**
+     * Control the maximum timeout in millis to wait for an invocation space to be available.
+     * <p/>
+     * If an invocation can't be made because there are too many pending invocations, then an exponential backoff is done
+     * to give the system time to deal with the backlog of invocations. This property controls how long an invocation is
+     * allowed to wait before getting a {@link com.hazelcast.core.HazelcastOverloadException}.
+     * <p/>
+     * <p>
+     * When set to -1 then <code>HazelcastOverloadException</code> is thrown immediately without any waiting.
+     * </p>
+     */
+    public static final HazelcastProperty INVOCATION_BACKOFF_TIMEOUT_MILLIS
```

Comment:
the question is whether to 1. align it with the server or 2. align it with the existing client properties.